robottelo/constants/__init__.py

```diff
@@ -39,11 +39,9 @@
 MAJOR_VERSION_DISTRO = {value: key for key, value in DISTROS_MAJOR_VERSION.items()}
 
 BROKER_DEPLOY_WORKFLOW = "deploy-base-rhel"
-BROKER_PROVISIONING_PROVIDER = "RHEV"
 BROKER_RHEL77 = {
     "workflow": BROKER_DEPLOY_WORKFLOW,
     "rhel_version": "7.7",
-    "provider": BROKER_PROVISIONING_PROVIDER,
```

Comment:
This change will impact all tests using the broker fixtures, and is necessary due to changes in AnsibleTower that broker is interfacing with.

Good work @swadeley !

Reply:
Hello, I decided to make a separate PR for this change: https://github.com/SatelliteQE/robottelo/pull/8453